projects/demo/src/modules/experimental/progress-segmented/examples/5/index.less

```diff
@@ -0,0 +1,5 @@
+[tuiProgressBar]._segmented {
+    --tui-segment-gap: 0.25rem;
+    border-radius: 0;
```

Comment:
Does it matter?

Reply:
https://github.com/taiga-family/taiga-ui/assets/35179038/1a1868e5-8d96-4513-9bf7-8b6fba3f0659